scripts/automation/tests/run.py

```diff
@@ -40,5 +38,21 @@ def run_all():
 
     return passed
 
+
 if __name__ == '__main__':
-    sys.exit(0 if run_all() else 1)
+    parse = argparse.ArgumentParser('Test tools')
+    parse.add_argument('--module', dest='modules', action='append',
+                       help='The modules of which the test to be run. Accept short names, except '
+                            'azure-cli, azure-cli-core and azure-cli-nspkg')
+    parse.add_argument('--non-parallel', action='store_true',
```

Comment:
It would also be nice to specify individual tests to run. Right now I have to do something like:

`python -m unittest azure.cli.command_modules.vm.tests.test_vm_commands.VMSSCreateNoneOptionsTest`

Which is kind of annoying because I have a TXT document with a bunch of these saved so I can just paste them into the command line. I'd much rather be able to just do:

`python -m automation.tests.run_tests --test test_vmss_create_none_options`